packages/form/tests/integration/components/cf-field/label-test.js

```diff
@@ -56,4 +56,39 @@ module("Integration | Component | cf-field/label", function (hooks) {
 
     assert.dom("label").hasText("Test (Optional)");
   });
+
+  test("it marks mandatory fields as such if useMandatoryAsterisk is configured in attribute", async function (assert) {
+    assert.expect(2);
+    this.set("useMandatoryAsterisk", true);
+
+    await render(
+      hbs`<CfField::Label
+  @field={{this.field}}
+  @useMandatoryAsterisk={{this.useMandatoryAsterisk}}
```

Comment:
```suggestion

    await render(
      hbs`<CfField::Label
  @field={{this.field}}
  @useMandatoryAsterisk={{true}}
```

You can just pass the value directly as `this.useMandatoryAsterisk` doesn't change in this test anyways. I feel like this is much easier to read as you don't have to search for that assignment this way.